algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -347,7 +358,7 @@ class CmsTotalCountProperty[K: CMSHasher: Gen] extends CmsProperty[K] {
 class CmsProperties extends ApproximateProperties("CountMinSketch") {
   import ApproximateProperty.toProp
 
-  implicit val intGen = Gen.choose(1, 100)
+  implicit val intGen = Gen.choose(Int.MinValue, Int.MaxValue)
```

Comment:
didn't we find that `Int.MinValue` was not good, or was it fine in the end?

actually, `(1, 100)` might be better to leave it since it will have collisions, which otherwise we aren't really exercising counting (since all counts will be 0 or 1 with an enormous range).